plugin/gv.vim

```diff
@@ -84,11 +84,19 @@ endfunction
 function! s:split(tab)
   if a:tab
     call s:tabnew()
-  elseif getwinvar(winnr('$'), 'gv')
-    $wincmd w
-    enew
   else
-    vertical botright new
+    let l:gv_win = 0
+    for l:win_number in range(1, winnr('$'))
+      if getwinvar(l:win_number, 'gv')
+        let l:gv_win = l:win_number
+      endif
+    endfor
+    if l:gv_win
+      call win_gotoid(win_getid(l:gv_win))
+    else
+      vertical botright new
+    end
+    enew
```

Comment:
Thanks, here's what I'm suggesting. Trying to minimize the code change.

```suggestion
    let w = get(filter(range(1, winnr('$')), 'getwinvar(v:val, "gv")'), 0)
    if w
      execute w.'wincmd w'
      enew
    else
      vertical botright new
    endif
```

* I don't use `l:` prefix in my code
* Any reason you decided to replace `wincmd w` with `win_gotoid + win_getid`?

Reply:
I see, thanks. I respect your coding convention of not using `l:`. 

I'm not a fan of using "exec" with string manipulation, but prefer using more explicit functions and APIs if applicable, but I'm fine with either. Your code with the functional API looks much better.